docs/sources/static/set-up/install-agent-macos.md

```diff
@@ -11,46 +11,58 @@ You can install Grafana Agent in static mode on macOS.
 
-## Install Grafana Agent with Homebrew
+## Installing Grafana Agent with Homebrew
 
-1. Open a terminal and run the following commands:
+Open a terminal and run the following commands:
 
-   ```
-   brew update
-   brew install grafana-agent
-   ```
+```shell
+brew update
+brew install grafana-agent
+```
 
-    The brew install command downloads Grafana Agent and installs it at `/opt/homebrew/Cellar/grafana-agent/VERSION`.
-    
-    Grafana Agent logs are in `/opt/homebrew/var/log/` by default.
+   Grafana Agent is installed by default at `$(brew --prefix)/Cellar/grafana-agent/VERSION`.
 
-1. Open a terminal and run the following commands:
+## Configuring Grafana Agent
 
-    ```
-    mkdir -p $(brew --prefix)/etc/grafana-agent/
+1. To create the Agent `config.yml` file, open a terminal and run the following command:
+
+    ```shell
     touch $(brew --prefix)/etc/grafana-agent/config.yml
     ```
 
-1. Modify `config.yml` with your configuration requirements.
+1. Edit `$(brew --prefix)/etc/grafana-agent/config.yml` and add the configuration blocks for your specific telemetry needs. Refer to [Configure Grafana Agent]({{< relref "../configuration/" >}}) for more information.
 
-    Refer to [Configure Grafana Agent]({{< relref "../configuration/" >}}) for information about the Agent configuration .
+{{% admonition type="note" %}}
+To send your data to Grafana Cloud, set up Grafana Agent using the Grafana Cloud integration. Refer to [how to install an integration](/docs/grafana-cloud/data-configuration/integrations/install-and-manage-integrations/) and [macOS integration](/docs/grafana-cloud/data-configuration/integrations/integration-reference/integration-macos-node/).
+{{%/admonition %}}
 
-1. Open a terminal and run the following command to start Grafana Agent:
+## Starting Grafana Agent
 
-    ` brew services start grafana-agent`
+Open a terminal and run the following command to start Grafana Agent:
 
-    For logs, see:
-    - stdout: `$(brew --prefix)/var/log/grafana-agent.log`
-    - stderr: `$(brew --prefix)/var/log/grafana-agent.err.log`
+```shell
+brew services start grafana-agent
+```
 
-1. Open a terminal and run the following command to upgrade Grafana Agent:
+## Viewing Grafana Agent Logs
 
-    `brew upgrade grafana-agent`.
+By default, logs are written to the following locations:
 
-{{% admonition type="note" %}}
-To send your data to Grafana Cloud, set up Grafana Agent using the Grafana Cloud integration. Refer to [how to install an integration](/docs/grafana-cloud/data-configuration/integrations/install-and-manage-integrations/) and [macOS integration](/docs/grafana-cloud/data-configuration/integrations/integration-reference/integration-macos-node/).
-{{%/admonition %}}
+* `$(brew --prefix)/var/log/grafana-agent.log`
+* `$(brew --prefix)/var/log/grafana-agent.err.log`
+
+## Upgrading Grafana Agent
+
+Open a terminal and run the following command to upgrade Grafana Agent:
+
+```shell
+brew upgrade grafana-agent
+ ```
```

Comment:
In my experience, you have to restart the service after upgrading, making the instructions:

```shell
brew upgrade grafana-agent 
brew services restart grafana-agent
```

Reply:
Updated the step